pkg/resources/metrics.go

```diff
@@ -78,6 +78,114 @@ func CreatePostgresConnectivityAlert(ctx context.Context, client k8sclient.Clien
 	return pr, nil
 }
 
+// CreatePostgresResourceStatusPhasePendingAlert creates a PrometheusRule alert to watch for Postgres CR state
+func CreatePostgresResourceStatusPhasePendingAlert(ctx context.Context, client k8sclient.Client, inst *v1alpha1.RHMI, cr *crov1.Postgres) (*prometheusv1.PrometheusRule, error) {
+	if strings.ToLower(inst.Spec.UseClusterStorage) == "true" {
+		logrus.Info("skipping postgres state alert creation, useClusterStorage is true")
+		return nil, nil
+	}
+	productName := cr.Labels["productName"]
+	postgresCRName := strings.Title(strings.Replace(cr.Name, "postgres-example-rhmi", "", -1))
+	alertName := postgresCRName + "PostgresResourceStatusPhasePending"
+	ruleName := fmt.Sprintf("resource-status-phase-pending-rule-%s", cr.Name)
+	alertExp := intstr.FromString(
+		fmt.Sprintf("absent(%s{exported_namespace='%s',resourceID='%s',productName='%s',statusPhase='complete'} == 1)",
```

Comment:
Now that we have the KSM-style alerts, could

```
statusPhase='in progress'} == 1
```

work here for pending checks?

Reply:
Good question we have a number of phases, 
https://github.com/integr8ly/cloud-resource-operator/blob/master/pkg/apis/integreatly/v1alpha1/types/types.go#L8-L16

and just tracking `in progress` may leave us not tracking other phases. The absence of  `statusPhase='complete'` would be the only phase that is not pending.

Reply:
This makes sense to me, thanks